doc/rst/dreln.1.rst

```diff
@@ -0,0 +1,63 @@
+dreln
+=====
+
+SYNOPSIS
+--------
+
+**dreln [OPTION] OLDPATH NEWPATH PATH ...**
+
+DESCRIPTION
+-----------
+
+Parallel MPI application to recursively update symlinks within a
+directory.
+
+dreln walks the specified PATH and updates any symlink whose target
+includes an absolute path to OLDPATH and replaces that symlink
+with a new link whose target points to NEWPATH instead.
+
+This is useful to update symlinks after migrating a large
+directory from one file system to another, whose links specify
+absolute paths to the original file system.
+
+OPTIONS
+-------
+
+.. option:: -i, --input FILE
+
+   Read source list from FILE. FILE must be generated by another tool
+   from the mpiFileUtils suite.
+
+.. option:: -r, --relative
+
+   Replace links using target paths that are relative to NEWPATH.
+
+.. option:: -v, --verbose
+
+   Run in verbose mode.
+
+.. option:: -h, --help
+
+   Print a brief message listing the :manpage:`drm(1)` options and usage.
+
+EXAMPLES
+--------
+
+1. To update all links under /new/path whose targets point to /orig/path
+and replace them with targets that point to /new/path:
+
+``mpirun -np 128 dreln -v /orig/path /new/path /new/path``
```

Comment:
using `/new/path`  twice in these examples is a little confusing. Maybe use `/original/link/path` `/new/link/path` and `/search/path`?

Reply:
Good point.  Updated now.